docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,71 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and token usage for Ubuntu Pro
+*********************************************
+
+Ubuntu Pro tokens are used to authenticate your machine and enable access to the services you are entitled to. tokens are unique and specific to your account, so you should treat them as a secret (as you would with a password).
+
+Since tokens are attached to your account and not your machine, you can re-use tokens, even multiple times per day. In this page, we'll discuss how tokens work and how machines are counted against your subscription.
+
+Pro tokens
+----------
+
+- Obtaining tokens:
+    After subscribing to Ubuntu Pro, you can access your tokens through the `Ubuntu Pro Dashboard <Pro_dashboard_>`_. Each token can be used to attach multiple machines depending on the subscription.
+
+- Attaching tokens:
+    Command: Use ``sudo pro attach`` to connect a machine to your Ubuntu Pro subscription.
+
+- Detaching tokens:
+    Command: Use ``sudo pro detach`` to remove a machine from the subscription. Once a machine is detached, the token can be reused to attach another machine. If you are attached on a VM you intend to delete, the token does not need to be detached before the VM is destroyed.
+
+For further information regarding pro commands, refer to our :ref:`Getting started-with Ubuntu Pro client tutorial<tutorial-commands>`.
+
+Monitoring active machines:
+---------------------------
+
+Active machines are counted based on the number of machines attached to your subscription at a given time.
+
+- Machine Limits: 
+    Each subscription type has a limit on the number of machines it can cover. For personal subscriptions, this limit is 5 physical machines with unlimited VMs on these machines. For enterprise subscriptions, the limit is based on the number of physical hosts and VMs. 
```

Comment:
```suggestion
    Each subscription type has a limit on the number of machines it can cover. For personal subscriptions, this limit is 5 physical machines with unlimited VMs on these machines. Enterprise subscriptions will have a higher limit and may differentiate between physical and virtual machines.
```